CMakeLists.txt

```diff
@@ -55,6 +55,12 @@ endif()
 if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   # using regular Clang or AppleClang
   set(LLVM_CLANG true)
+  if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "10.0.0" AND
+     CMAKE_SYSTEM_NAME STREQUAL "Linux")
+    message(WARNING "Found LLVM < 10 which has known optimization bugs with \
+    certain glibc versions. In case your build fails with linking errors. \
+    Use -DFAST_MATH=off - See https://bugs.llvm.org/show_bug.cgi?id=45034")
```

Comment:
Disabling fast math is a bad idea. I guess the resulting binary is uselessly slow when using EQs.
Can't we just recommend to use gcc?

Reply:
I changed the suggestion to prefer the use gcc and only in extrem cases disable fast math